src/firefox/index.js

```diff
@@ -127,13 +128,13 @@ export async function run(
   {
     fxRunner = defaultFxRunner,
     findRemotePort = defaultRemotePortFinder,
-    firefoxBinary, binaryArgs,
+    firefoxBinary, binaryArgs, port,
   }: FirefoxRunOptions = {}
 ): Promise<FirefoxProcess> {
 
   log.debug(`Running Firefox with profile at ${profile.path()}`);
 
-  const remotePort = await findRemotePort();
+  const remotePort = port || await findRemotePort();
```

Comment:
Huh. I just realized that the call to `await findRemotePort()` means that Firefox was always starting up with a unique remote debugger port. Maybe all we need to do is return the port? Currently it returns the `firefox` instance so maybe it needs to return `{firefox, port}` instead.